lib/install.ps1

```diff
@@ -1220,3 +1222,31 @@ function persist_data($manifest, $original_dir, $persist_dir) {
         }
     }
 }
+
+#check whether write permission for Users usergroup is set to global persist dir, if not then set
+function persist_permission($persist, $global) {
+    if ($persist -and $global) {
+        $path = "$(basedir $global)\persist"
+        $user = -join ([System.Environment]::MachineName, '\',
+            (([System.Security.Principal.SecurityIdentifier]'S-1-5-32-545').Translate([System.Security.Principal.NTAccount])).Value)
```

Comment:
This would result in: `R15CH13-PC\VORDEFINIERT\Benutzer`

Reply:
The important part is "testing"
I just installed and uninstalled notepad++ with these changes about 30 times now 😁